### PR TITLE
Date received all items for application and NOI

### PR DESCRIPTION
**Pull request by lstod**
Note this will not work with obfuscated data, to test either remove the line for 'AND proxy' or use a different who_created value in the DB

### CODE REVIEW

File: bin/migrate-oats-data/applications/migrate_application.py [hunk 1]
```diff
@@ -15,6 +16,7 @@
 
 def process_application_etl(batch_size):
     process_alcs_application_prep_fields(batch_size)
+    update_application_date_rx_all_items(batch_size)
```
Comment (mhuseinov) [inline bin/migrate-oats-data/applications/migrate_application.py:19]: Does 'rx' stand for 'received'?

File: bin/migrate-oats-data/applications/sql/application-prep/app_rx_all_items.sql [hunk 2]
```diff
@@ -0,0 +1,11 @@
+SELECT
```
Comment (mhuseinov) [inline bin/migrate-oats-data/applications/sql/application-prep/app_rx_all_items.sql:1]: This query will duplicate applications on joining the accomplishments is that expected?

File: bin/migrate-oats-data/applications/migrate_application.py (hunk 1 above)
Comment (lstod) [reply bin/migrate-oats-data/applications/migrate_application.py:19]: correct

File: bin/migrate-oats-data/applications/sql/application-prep/app_rx_all_items.sql (hunk 2 above)
Comment (lstod) [reply bin/migrate-oats-data/applications/sql/application-prep/app_rx_all_items.sql:1]: I believe it is intened in order to capture both AKC and SAL

**Review by mhuseinov — APPROVED** [review]

**Pull request merged**